pkgs/jnigen/lib/src/bindings/renamer.dart

```diff
@@ -94,20 +90,21 @@ const Map<String, int> _definedSyms = {
   'release': 1,
   'releasedBy': 1,
   'jClass': 1,
+  'type': 1,
 };
 
-/// Appends 0 to [name] if [name] is a keyword.
+/// Appends $ to [name] if [name] is a keyword.
 ///
 /// Examples:
-/// * `int` -> `int0`
+/// * `int` -> `int$`
```

Comment:
Not a good example, since `int` is not a Dart keyword, and is also not in the `keywords` set.

If this is doing something other than renaming Dart reserved words or built-in identifiers, then do say what. (If not, the example is wrong.)

Reply:
Already merged, will address in a another PR.